source/agora/common/crypto/Key.d

```diff
@@ -123,7 +132,7 @@ public struct PublicKey
     /// Uses Stellar's representation instead of hex
     public string toString () const @trusted nothrow
     {
-        ubyte[1 + PublicKey.Width + 2] bin;
+        ubyte[1 + PublicKey.sizeof + 2] bin;
```

Comment:
Perhaps descriptive constants could be used for these extra byte lengths.

Reply:
I see what you mean. I will add them in another commit.